src/app/templates/SeedPhraseInput/index.tsx

```diff
@@ -48,9 +57,13 @@ export const SeedPhraseInput: FC<SeedPhraseInputProps> = ({
       let newSeedError = '';
       const joinedDraftSeed = newDraftSeed.join(' ');
 
-      if (newDraftSeed.some(word => word !== '')) {
+      if (newDraftSeed.some(Boolean)) {
         if (newDraftSeed.some(word => word === '')) {
-          newSeedError = t('mnemonicWordsAmountConstraint');
+          if (newDraftSeed.some(word => word !== '' && bip39WordList.includes(word))) {
+            newSeedError = '';
+          } else {
+            newSeedError = t('mnemonicWordsAmountConstraint', [numberOfWords]) as string;
+          }
         } else if (!validateMnemonic(formatMnemonic(joinedDraftSeed))) {
           newSeedError = t('justValidPreGeneratedMnemonic');
         }
```

Comment:
Try creating a variable like `const draftSeedHasValidWord = newDraftSeed.some(word => word !== '' && bip39WordList.includes(word));` and flattening these conditions